x/gateway/keeper/params_test.go

```diff
@@ -16,3 +20,41 @@ func TestGetParams(t *testing.T) {
 	require.NoError(t, k.SetParams(ctx, params))
 	require.EqualValues(t, params, k.GetParams(ctx))
 }
+
+func TestParams_ValidateMinRelayDifficulty(t *testing.T) {
```

Comment:
```suggestion
func TestParams_ValidateMinStake(t *testing.T) {
```

Reply:
I expect this sort of copypasta to stop once I can start using the updated "adding params" doc. :sweat_smile: 

Thanks for catching these! :raised_hands: